### PR TITLE
Add django-2.0 to tox matrix, update requirements, bump version

### PR DESCRIPTION
**Pull request by dekoza**

### CODE REVIEW

File: testproject/settings.py [hunk 1]
```diff
@@ -19,10 +22,11 @@
 
 SECRET_KEY = '_'
 
-MIDDLEWARE = [
+MIDDLEWARE_CLASSES = MIDDLEWARE = [
```
Comment (haxoza) [inline testproject/settings.py]: Do we really need `MIDDLEWARE_CLASSES`?

File: testproject/settings.py [hunk 2]
```diff
@@ -1,6 +1,9 @@
 from __future__ import absolute_import
 import os
+import sys
 
+sys.path.append('../')
+sys.path.append('./')
```
Comment (haxoza) [inline testproject/settings.py]: Why?

File: README.rst [hunk 3]
```diff
@@ -35,8 +35,8 @@ Requirements
 To be able to run **djoser** you have to meet following requirements:
 
 - Python (2.7, 3.4, 3.5, 3.6)
-- Django (1.10, 1.11)
-- Django REST Framework (3.7)
+- Django (1.11, 2.0)
```
Comment (haxoza) [inline README.rst]: 1.10 is removed from supported versions in README but I can see it in tox dependencies.

File: .gitignore [hunk 4]
```diff
@@ -1,40 +1,197 @@
...
+/site
+
+# mypy
+.mypy_cache/
+
+### Python Patch ###
+.venv/
+
+
+# End of https://www.gitignore.io/api/django,python,pycharm+all
```
Comment (haxoza) [inline .gitignore]: I'm not the fan of auto generated ignore file for libraries like that. I believe we will never ever use e.g. crashlitics in this project 😄

Comment (dekoza) [reply .gitignore]: I'm more for standards but I admit the .gitignore here might be overkill :wink:

**Comment by dekoza** [comment]
Actually, I managed to update urls to more RESTful fashion, so this PR is no longer needed. Changes will be included in next PR with said urls.

**Pull request closed without merging**